openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```diff
@@ -585,42 +586,42 @@
     "length-validator-error": "At least {{length}} {{field}} required",
     "lineage-data-is-not-available-for-deleted-entities": "Lineage data is not available for deleted entities.",
     "list-of-strings-regex-patterns-csv": "Enter a list of strings/regex patterns as a comma separated value",
-    "make-an-announcement": "Make an announcement",
+    "make-an-announcement": "Make an announcement!",
     "markdown-editor-placeholder": "Use @mention to tag a user or a team.\nUse #mention to tag a data asset.",
     "mentioned-you-on-the-lowercase": "mentioned you on the",
-    "most-active-users": "Displays the most active users on the platform based on page views.",
+    "most-active-users": "Displays the most active users on the platform based on Page Views.",
     "most-viewed-data-assets": "Displays the most viewed data assets.",
     "new-to-the-platform": "New to the platform?",
-    "no-data-available": "No data available",
+    "no-data-available": "No data available.",
     "no-data-available-for-selected-filter": "No data found. Try changing the filters.",
     "no-execution-runs-found": "No execution runs found for the pipeline.",
     "no-features-data-available": "No features data available",
     "no-ingestion-available": "No ingestion data available",
-    "no-ingestion-description": "To view Ingestion Data, run the MetaData Ingestion. Please refer to this doc to schedule the",
+    "no-ingestion-description": "To view Ingestion Data, run the metadata ingestion. Please refer to this doc to schedule the",
```

Comment:
incomplete sentence

Reply:
This is used with other keys to make it a complete sentence.